configs/analysis_configs/consensus_configs.yaml

```diff
@@ -0,0 +1,13 @@
+consensus_config:
```

Comment:
hmm, I'm still a bit confused by these parameters. Where did you take them from?

Shouldn't the parameters for consensus (like the other steps) include only the arguments in [`pycytominer.consensus`](https://github.com/cytomining/pycytominer/blob/master/pycytominer/consensus.py)?

Reply:
this came from the aggregate function: https://github.com/cytomining/pycytominer/blob/master/pycytominer/aggregate.py#L14

but I will replace it! I didn't know there was a consensus function